config/consts.rb

```diff
@@ -102,6 +102,9 @@ def initialize
   config.location_bad_terms_file = "#{location_path}bad_terms.yml"
   config.unknown_location_name = "Earth"
   config.obs_location_max_area = 24_000
+  # Geographic epsilon for bounding box comparisons (~11 meters at equator).
+  # Used for: rounding tolerance, point vs box threshold, minimal box size.
+  config.box_epsilon = 0.0001
```

Comment:
Question (for my own benefit): When do constants get defined here vs in a model or elsewhere?

Reply:
For me, it's when they are needed everywhere: classes, models, tests. 

This is probably most properly a constant of `Mappable::Box`, because if Locations became polygons, it would go away. 

But seems ok as an MO constant.